<launch>

    <!-- Configurable arguments -->
    <arg name="robot_name" default="lolo"/>
    <arg name="system_name" default="$(arg robot_name)"/>
	<!-- Croatia, biograd coast, lat lon zone band, all four from there -->
    <arg name="latitude" default="59.339363"/>
    <arg name="longitude" default="18.159550"/>
    <arg name="utm_zone" default="34"/>
    <arg name="utm_band" default="V"/>

    <arg name="base_frame" default="$(arg robot_name)/base_link"/>
    <arg name="goal_tolerance" default="5."/>

    <group ns="$(arg robot_name)">

        <!--
        <group ns="ctrl">
            <node name="yaw_planner" pkg="auv_simple_motion_planner" type="yaw_planner.py" output="screen">
                <param name="goal_tolerance" value="$(arg goal_tolerance)" />
                <param name="base_frame" value="$(arg base_frame)" />
            </node>
        </group>
        -->

        <node name="sam_bt" pkg="waypoints" type="sam_bt.py" output="screen">
            <param name="system_name" value="$(arg system_name)" />
            <param name="utm_zone" value="$(arg utm_zone)" />
            <param name="utm_band" value="$(arg utm_band)" />
            <remap from="/bezier_planner" to="ctrl/yaw_planner"/>
        </node>   

    </group>
    
</launch>
